modules/weko-deposit/weko_deposit/api.py

```diff
@@ -1074,6 +1074,11 @@ def get_record_by_pid(cls, pid):
         pid = PersistentIdentifier.get('depid', pid)
         return cls.get_record(id_=pid.object_uuid)
 
+    @classmethod
+    def get_record_by_id(cls, idx):
+        """Get record by id."""
+        return cls.get_record(id_=idx)
+
```

Comment:
This method is redundant, you can call the method `get_record()` directly.
Example
```python
WekoRecord.get_record(record['id'])
```